src/CfgEditor/CfgEditorPanel.ts

```diff
@@ -88,7 +88,7 @@ export class CfgEditorPanel implements vscode.CustomTextEditorProvider {
     webviewPanel.webview.html = await this._getHtmlForWebview(webviewPanel.webview);
 
     const changeDocumentSubscription = vscode.workspace.onDidChangeTextDocument(e => {
-      if (e.document.uri.toString() === document.uri.toString()) {
+      if (e.contentChanges.length > 0 && e.document.uri.toString() === document.uri.toString()) {
```

Comment:
Hmm... if `onDidChangeTextDocument can be called even when there are no changes.`, can we be sure `e` is not null? (99% it is not null)

Reply:
Q. `e.document.uri.toString() === document.uri.toString()` can be replaced by `e.document.uri === document.uri`?

Reply:
@YongseopKim I tested uri comparison before, and URIs with the same paths were not the same by === operator....

Reply:
> can we be sure e is not null? (99% it is not null)

I don't know the exact implementation of VSCode internal logic, but I am sure because VSCode sample extension is written exactly same :D

> Q. e.document.uri.toString() === document.uri.toString() can be replaced by e.document.uri === document.uri?

I think this can be dangerous because `object === object` compares their memory address, not member variables..!